In [8]:
#入力受け取り
print("input N >> ",end = "")
N = int(input())
print(N)

#ファイルの読み込み、1行ずつ配列に格納
f = open("popular-names.txt","r")
content = f.read().splitlines()          #['Mary\tF\t7065\t1880', 'Anna\tF\t2604\t1880',...]

#表示に関わる処理 
#表示する範囲は[-N,-1]になるようにループ範囲を調整
for i in range(N,0,-1):
    print(content[-i])

f.close()

input N >> 5
Benjamin	M	13381	2018
Elijah	M	12886	2018
Lucas	M	12585	2018
Mason	M	12435	2018
Logan	M	12352	2018


--UNIXコマンドによる確認画面--
・-n(op) 表示する行数を指定

shota@LAPTOP-7H1S837B:~/py_venvs/nlp100/100knock2024/shota/chapter02$ tail -n 5 popular-names.txt
Benjamin        M       13381   2018
Elijah  M       12886   2018
Lucas   M       12585   2018
Mason   M       12435   2018
Logan   M       12352   2018